ansible_base/lib/testing/fixtures.py

```diff
@@ -50,7 +50,7 @@ def local_authenticator(db):
         name="Test Local Authenticator",
         enabled=True,
         create_objects=True,
-        remove_users=True,
+        remove_users=False,
```

Comment:
when `True` this breaks some rbac tests. Because it used to remove only member roles, now it removes all user roles

Reply:
This isn't totally clear to me. There's an `org_admin_rd` fixture. That fixture by itself just gives the role definition and shouldn't have any connection to claims processing. If you do claims processing with an empty roles list, then sure, it might remove org admin permissions, but I don't know where that would be a problem.

Reply:
bad example. It's failing i.e. on 
```
FAILED tests/rbac/api/test_user_permissions.py::TestUserDetailView::test_superuser_can_delete_new_user[True] - assert 403 == 204
```

I think the issue is in the `admin_api_client` + `admin_user` fixtures. 
My guess is that the `admin_user` is created with flag `is_superuser`, then `admin_api_client` calls `login`, which call the `reconcile_user_claims`, which removes the `is_superuser` flag